.github/workflows/ci.yml

```diff
@@ -101,7 +91,6 @@ jobs:
         uses: "ramsey/composer-install@v2"
```

Comment:
Version 3 was released this week. This could be changed to fix node16 => 20 deprecation in ci